python/raydp/context.py

```diff
@@ -56,13 +60,15 @@ def __init__(self,
                  num_executors: int,
                  executor_cores: int,
                  executor_memory: Union[str, int],
+                 enable_hive: Optional[bool],
```

Comment:
No need to use `Optional` here, because it's never None, just False or True.

Reply:
yes